src/main/app/models/draft.ts

```diff
@@ -0,0 +1,11 @@
+import { Moment } from 'moment'
+
+import { DraftDocument } from 'models/draftDocument'
+
+export class Draft<T extends DraftDocument> {
+  constructor (public id: number = undefined,
+               public type: string,
+               public document: T,
+               public created?: Moment,
```

Comment:
is created optional?

Reply:
did you using separate models for reads and writes?